pkg/asset/machines/worker.go

```diff
@@ -176,20 +176,20 @@ func defaultPowerVSMachinePoolPlatform() powervstypes.MachinePool {
 
 func defaultNutanixMachinePoolPlatform() nutanixtypes.MachinePool {
 	return nutanixtypes.MachinePool{
-		NumCPUs:           2,
-		NumCoresPerSocket: 2,
-		MemoryMiB:         8192,
+		NumCPUs:           4,
+		NumCoresPerSocket: 4,
```

Comment:
This implies 16 CPU units (CPUs x Cores per CPU). Is that what we want? I actually have an [open PR that was reducing these even further.](https://github.com/openshift/installer/pull/5857)

Reply:
I just moved the code from [1] and the minimum requirement is 2 and 2 but the vsphere folks set it to 4 and 4 as part of the performance recommendation for vsphere. I'll remove the nutanix change though to let you change it to the ideal value, thanks!

Reply:
@rna-afk I can also take care of bumping up the RAM to 16GB in my PR.